gempython/tools/amc_user_functions_xhal.py

```diff
@@ -317,12 +317,12 @@ def performDacScanMultiLink(self, dacDataAll, dacSelect, dacStep=1, ohMask=0x3ff
             exit(os.EX_USAGE)
 
         # Check length of results container
-        lenExpected = nUnmaskedOHs * (maxVfat3DACSize[dacSelect][0] - 0+1)*24 / dacStep
+        lenExpected = self.nOHs * (maxVfat3DACSize[dacSelect][0] - 0+1)*24 / dacStep
         if (len(dacDataAll) != lenExpected):
             print("HwAMC::performDacScanMultiLink(): I expected container of lenght {0} but provided 'dacDataAll' has length {1}",format(lenExpected, len(dacDataAll)))
```

Comment:
as long as this line (322) shows up in the diff, may as well fix the typo `lenght`